app/assets/javascripts/admin/templates/group.hbs

```diff
@@ -106,6 +106,7 @@
     {{#unless model.automatic}}
       <button {{action "destroy"}} class='btn btn-danger'>{{fa-icon "trash-o"}}{{i18n 'admin.customize.delete'}}</button>
     {{/unless}}
+    <span {{bind-attr class=":saving savingStatus::hidden"}}>{{savingStatus}}</span>
```

Comment:
`bind-attr` is deprecated in future ember releases. I can accept this but in the future we should avoid it :)